runner/organizations_test.go

```diff
@@ -22,19 +22,70 @@ import (
 	dbMocks "garm/database/common/mocks"
 	runnerErrors "garm/errors"
 	"garm/params"
+	"garm/runner/common"
+	poolManager "garm/runner/common/mocks"
+	providerMocks "garm/runner/common/mocks"
+	runnerMocks "garm/runner/mocks"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
+func TestCreateOrganization(t *testing.T) {
+	adminCtx := auth.GetAdminContext()
+	createOrgParams := params.CreateOrgParams{
```

Comment:
Setting up the dummy credentials and mocks feels a bit repetitive. Can we create a ```SetupTest()``` function or something similar?

```testify``` has the option to define a ```suite``` which gives you a ```SetupTest()``` function. This also allows you to create your own ```type``` and populate it on ```SetupTest()``` with any fixtures you may need, like credentials, responses, whatever makes sense.

You can use this to de-duplicate code.

https://github.com/stretchr/testify#suite-package

Reply:
Sounds good. I will look to remove the de-duplicate code.